docs/rest_spec.txt

```diff
@@ -0,0 +1,70 @@
+REST specification 
+
+An email is compared to cattle and the analogy is used throughout. Sometimes, stretched to the point of it being nauseating! Blame Omkar Ekbote for it.
+
+POST on fodder: 
+  Input: Data stream of donor details. (Ideally an excel sheet) 
+  Output: 200/err_string
+
+GET on fodder/ingredients: Get headers from excel 
+  Input: None 
+  Output: List of headers
+  Test: curl -i http://localhost:5000/api/fodder/ingredients
+HTTP/1.0 200 OK
+Content-Type: application/json
+Content-Length: 148
+ContentType: application/json
+Access-Control-Allow-Origin: *
+Server: Werkzeug/0.14.1 Python/2.7.5
+Date: Thu, 08 Nov 2018 12:49:30 GMT
+
+["S.No","Name of Child","Class","Sponsor","Reference","Sponsor Mail Id","Reference Mail Idemail subject","email body","attachment","has_sent_mail"]
```

Comment:
Is this a copy-paste error or did 2 heads get smashed into each other?

Reply:
This issue is due to an error in py file. Have resolved it.